integration/spark2/src/main/scala/org/apache/spark/sql/execution/CastExpressionOptimization.scala

```diff
@@ -21,12 +21,14 @@ import java.text.{ParseException, SimpleDateFormat}
 import java.util
 import java.util.{Locale, TimeZone}
 
+import scala.Option
 import scala.collection.JavaConverters._
 
-import org.apache.spark.sql.catalyst.expressions.{Attribute, Cast, EmptyRow, EqualTo, Expression, GreaterThan, GreaterThanOrEqual, In, LessThan, LessThanOrEqual, Literal, Not}
+import org.apache.spark.sql.catalyst.expressions.{Attribute, EmptyRow, EqualTo, Expression, GreaterThan, GreaterThanOrEqual, In, LessThan, LessThanOrEqual, Literal, Not}
 import org.apache.spark.sql.CastExpr
 import org.apache.spark.sql.sources
-import org.apache.spark.sql.types.{DoubleType, IntegerType, StringType, TimestampType}
+import org.apache.spark.sql.types.{DataType, DoubleType, IntegerType, StringType, TimestampType}
+import org.apache.spark.sql.CarbonExpressions.{MatchCast => Cast}
```

Comment:
Don't change the other imports, there are unnecessary imports here

Reply:
Ok